tests/gen/Human.hack

```diff
@@ -34,6 +35,39 @@ final class Human extends \Slack\GraphQL\Types\ObjectType {
           dict[],
           async ($parent, $args, $vars) ==> $parent->getFavoriteColor(),
         );
+      case 'friends':
+        return new GraphQL\FieldDefinition(
+          'friends',
+          UserConnection::nullableOutput(),
+          dict[
+            'after' => shape(
+              'name' => 'after',
+              'type' => Types\StringType::nullableInput(),
+              'default_value' => null,
+            ),
+            'before' => shape(
+              'name' => 'before',
+              'type' => Types\StringType::nullableInput(),
+              'default_value' => null,
+            ),
+            'first' => shape(
+              'name' => 'first',
+              'type' => Types\IntType::nullableInput(),
+              'default_value' => null,
+            ),
+            'last' => shape(
+              'name' => 'last',
+              'type' => Types\IntType::nullableInput(),
+              'default_value' => null,
+            ),
+          ],
+          async ($parent, $args, $vars) ==> (await $parent->getFriends())->setPaginationArgs(
```

Comment:
This is the special cased resolver generated by the `ConnectionFieldBuilder` method. Yuck, but maybe it's OK? 🤔

Reply:
Looks fine to me, why yuck? :)

You might want to add a test field that has extra arguments, so we will have `coerce` calls here both inside `getFriends` and inside `setPaginationArgs`.

Reply:
Just feels yucky to have one special case among code that is generally not full of special cases.

That said, good idea on that test case. Will add.